go.mod

```diff
@@ -18,15 +18,14 @@ require (
 	github.com/dgraph-io/ristretto v0.1.0
 	github.com/golang/protobuf v1.5.2
 	github.com/hashicorp/go-uuid v1.0.2
-	github.com/hashicorp/golang-lru v0.5.4 // indirect
-	github.com/hyperledger-labs/fabric-smart-client v0.0.0-20220608063331-d079602e00f2
+	github.com/hyperledger-labs/fabric-smart-client v0.0.0-20220616081851-e7062aa93fc7
 	github.com/hyperledger-labs/orion-sdk-go v0.0.0-20220213082028-b0ee3d8f361c
 	github.com/hyperledger-labs/orion-server v0.2.1
 	github.com/hyperledger/fabric v1.4.0-rc1.0.20220128025611-fad7f691a967
 	github.com/hyperledger/fabric-chaincode-go v0.0.0-20210718160520-38d29fabecb9
 	github.com/hyperledger/fabric-protos-go v0.0.0-20210911123859-041d13f0980c
-	github.com/json-iterator/go v1.1.10
-	github.com/libp2p/go-libp2p-core v0.3.0
+	github.com/json-iterator/go v1.1.12
+	github.com/libp2p/go-libp2p-core v0.16.1
```

Comment:
do we actually need this direct dependency?

Reply:
FSC should take care about that

Reply:
yes, once we merge https://github.com/hyperledger-labs/fabric-smart-client/pull/294, I'll tidy up the dependencies :)

Reply:
okay, `go mod tidy` does not remove them :(